<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButton_del">
     <property name="text">
      <string>del</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_std">
     <property name="text">
      <string>standard</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_string">
     <property name="text">
      <string>string</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_file">
     <property name="text">
      <string>file</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QColumnView" name="columnView"/>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QListView" name="listView"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
